templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -578,6 +580,8 @@ export class {{ service.name }}Client {
 {%- if method.serverStreaming and method.clientStreaming %}
 /**
 {{- util.printComments(method, service) }}
+ * @example <caption>include:samples/generated/{{api.naming.version}}/{{service.name.toSnakeCase()}}.{{method.name.toSnakeCase()}}.js</caption>
```

Comment:
Seems `<caption></caption> ` not working in TSDoc
https://tsdoc.org/play

Reply:
I think this is for cloud rad documentation specifically, @fhinkel